db/scripts/init.sql

```diff
@@ -15,3 +15,15 @@ CREATE TABLE IF NOT EXISTS brothers(
 
 INSERT INTO brothers (pacificId, firstName, lastName, status, className, rollCall, email, phoneNumber, badStanding)
 VALUES (989123456, 'Nicolas', 'Ahn', 'Active', 'Chi', '000', 'na@gmail.com', '(123) 456-7890', 0);
+
+
+CREATE TABLE Events(
+  event_id INT PRIMARY KEY NOT NULL, 
+  event_name TEXT NOT NULL,
+  category INT NOT NULL,
```

Comment:
Need to create a "eventCategory" table

Reply:
I think it's best to figure out how we want to design eventCategory before adding it in this PR